### ISSUE TITLE
test_cache build failure

### ISSUE THREAD

**Issue by daniellowell** [ROCmSoftwarePlatform/MIOpen#258]
On the latest `develop` branch on ROCm 3.5 I am seeing this issue when building  

```
$ make -j test_cache
[  0%] Built target addkernels
[ 96%] Built target MIOpen
[100%] Building CXX object test/CMakeFiles/test_cache.dir/cache.cpp.o
/home/dlowell/MIOpen/test/cache.cpp:135:22: error: no member named 'GetCacheFile' in namespace 'miopen'
    auto p = miopen::GetCacheFile("gfx", "base", "args", false);
             ~~~~~~~~^
/home/dlowell/MIOpen/test/cache.cpp:141:25: error: no member named 'GetCacheFile' in namespace 'miopen'
    auto p    = miopen::GetCacheFile("gfx", "base", "args", true);
                ~~~~~~~~^
2 errors generated when compiling for gfx906.
test/CMakeFiles/test_cache.dir/build.make:62: recipe for target 'test/CMakeFiles/test_cache.dir/cache.cpp.o' failed
make[3]: *** [test/CMakeFiles/test_cache.dir/cache.cpp.o] Error 1
CMakeFiles/Makefile2:7590: recipe for target 'test/CMakeFiles/test_cache.dir/all' failed
make[2]: *** [test/CMakeFiles/test_cache.dir/all] Error 2
CMakeFiles/Makefile2:7597: recipe for target 'test/CMakeFiles/test_cache.dir/rule' failed
make[1]: *** [test/CMakeFiles/test_cache.dir/rule] Error 2
Makefile:2953: recipe for target 'test_cache' failed
make: *** [test_cache] Error 2
```

This happens on both backends. Checking multiple systems.

**Comment by daniellowell** [comment]
False alarm.

**Comment by atamazov** [comment]
This is a bug that I've introduced when switched from sqlite binary cache to file based one. Needs to be fixed, but not urgent.

**Comment by atamazov** [comment]
Fixed by #297
